specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/azureADOnlyAuthentication.json

```diff
@@ -0,0 +1,299 @@
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/azureADOnlyAuthentications": {
+      "get": {
+        "operationId": "AzureADOnlyAuthentications_List",
+        "summary": "Gets a list of Azure Active Directory only authentication property",
+        "description": "Gets a list of Azure Active Directory only authentication property for a workspace",
+        "tags": [
+          "WorkspaceAzureADOnlyAuthentications"
+        ],
+        "deprecated": false,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./azureADOnlyAuthentication.json#/parameters/workspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "schema": {
+              "$ref": "#/definitions/AzureADOnlyAuthenticationListResult"
+            },
+            "description": "OK"
+          },
+          "default": {
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            },
+            "description": ""
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Get a list of Azure Active Directory Only Authentication property": {
+            "$ref": "./examples/ListAzureADOnlyAuthentication.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AzureADOnlyAuthenticationProperties": {
+      "description": "Properties of a active directory only authentication.",
+      "required": [
+        "azureADOnlyAuthentication"
+      ],
+      "type": "object",
+      "properties": {
+        "azureADOnlyAuthentication": {
+          "description": "Azure Active Directory only Authentication enabled.",
+          "type": "boolean"
+        },
+        "state": {
+          "description": "property configuration state",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Consistent",
+            "InConsistent",
+            "Updating"
+          ],
+          "x-ms-enum": {
+            "name": "state value",
+            "modelAsString": true
+          }
+        },
+        "creationDate": {
```

Comment:
>creationDate [](http://example.com/codeflow?start=9&length=12)

This shouldn't be needed, since it's already included in systemData outside the property bag.

Reply:
We don't support system data in any of the Synapse RP APIs

Reply:
Will be removing this.

Reply:
Removed systemData